demos/pkcs11/pkcs11_demo_sign_and_verify/pkcs11_demo_sign_and_verify.c

```diff
@@ -155,7 +155,7 @@ CK_RV PKCS11SignVerifyDemo( void )
     {
         result = xFindObjectWithLabelAndClass( session,
                                                pkcs11demoPRIVATE_KEY_LABEL,
-                                               sizeof( pkcs11demoPRIVATE_KEY_LABEL ),
+                                               sizeof( pkcs11demoPRIVATE_KEY_LABEL ) - 1,
```

Comment:
Is this change to discount the `NUL` character? 
The comment in the `pkcs11_demo_objects.c` file mentions that the labels should not be NUL terminated...
https://github.com/aws/aws-iot-device-sdk-embedded-C/blob/a6ae98535d1f304830bae1a1cd8029887792868e/demos/pkcs11/pkcs11_demo_objects/pkcs11_demo_objects.c#L337-L340

Reply:
Yes, it is not NUL terminated in the objects demo as well because when passed to `publicKeyTemplate`/`privateKeyTemplate`, the NUL character is removed through the length.
https://github.com/aws/aws-iot-device-sdk-embedded-C/blob/a6ae98535d1f304830bae1a1cd8029887792868e/demos/pkcs11/pkcs11_demo_objects/pkcs11_demo_objects.c#L358
Hence, NUL character must also be subtracted from the length in the sign and verify demo.

Reply:
Okay, should the `It should not be NULL terminated.` comment be removed then as it is not fitting in declaration of the byte arrays..